pkgs/top-level/python-packages.nix

```
@@ -4318,6 +4318,7 @@ in {
   kmsxx = (callPackage ../development/libraries/kmsxx {
```

so that certain hooks work correctly
```suggestion
  kmsxx = toPythonModule (callPackage ../development/libraries/kmsxx {
```

---

@jonringer Done